src/App.test.tsx

```diff
@@ -3,9 +3,13 @@ import { shallow } from 'enzyme';
 
 import App from './App';
 import styles from './App.module.scss';
+import configureStore from './configureStore';
```

Comment:
If CRA let's us add them, sure. I'm not too worried about it since the tests will be co-located and I don't think the app will get as big as addons-frontend.